### PR TITLE
New client version should update docs repository

### PR DESCRIPTION
**Pull request by abhinayagarwal**
We update the docs repository whenever a new snapshot or release version of the client-plugin is released. There is no change made if the snapshot is build for the same version twice.

### CODE REVIEW

File: .ci/update-docs.sh [hunk 1]
```diff
@@ -9,10 +9,12 @@ cd docs
...
-s3cmd --no-mime-magic --guess-mime-type --access_key "$AWS_ACCESS_KEY" --secret_key "$AWS_SECRET_KEY" sync -P build/docs/html5/client/ s3://docs.gluonhq.com/client/$1/
\ No newline at end of file
+# Commit only if there are changes
+git diff --quiet && git diff --staged --quiet
+RESULT=$?
+if [ $RESULT -eq 0 ]; then
+  echo "There are no changes to commit"
+else
+  git commit gradle.properties -am "Update client version to v$1"
+  git push https://gluon-bot:$GITHUB_PASSWORD@github.com/$SAMPLES_REPO_SLUG HEAD:master
```
Comment (jperedadnr) [inline .ci/update-docs.sh:19]: SAMPLES_REPO? Shouldn't it be docs repo?

**Review by jperedadnr — APPROVED** [review]

**Comment by jperedadnr** [comment]
@abhinayagarwal Is this PR still needed?

**Comment by abhinayagarwal** [comment]
It is certainly out of date. It needs a few updates, but definitely required to keep docs upto date with latest plugin version.

**Comment by jperedadnr** [comment]
Still the case?

**Comment by abhinayagarwal** [comment]
Closing, as no longer required.

**Pull request closed without merging**